one_fm/overrides/employee.py

```diff
@@ -278,11 +299,34 @@ def notify_authorities(self):
                 sendemail(recipients=the_recipient, subject=title, content=msg)
         except Exception as e:
             frappe.log_error(frappe.get_traceback(), "Error while sending mail on status change(Employee)")
-        
-        
-        
-        
-        
-        
-        
-    
\ No newline at end of file
+
+@frappe.whitelist()
+def get_employee_reports_to_user(employee):
```

Comment:
Hey @pjamsheer, We have a function: one_fm.api.tasks import get_action_user, which fetches the reports to an employee. Could you please check if that could be reused here?

Reply:
In get_action_user the shift is mandatory and there are employees without shift allocation